ProcessLib/SmallDeformationNonlocal/CreateSmallDeformationNonlocalProcess.cpp

```diff
@@ -0,0 +1,141 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateSmallDeformationNonlocalProcess.h"
+
+#include <cassert>
+
+#include "MaterialLib/SolidModels/CreateConstitutiveRelation.h"
+#include "ProcessLib/Output/CreateSecondaryVariables.h"
+#include "ProcessLib/Utils/ProcessUtils.h"
+
+#include "SmallDeformationNonlocalProcess.h"
+#include "SmallDeformationNonlocalProcessData.h"
+
+namespace ProcessLib
+{
+namespace SmallDeformationNonlocal
+{
+template <int DisplacementDim>
+std::unique_ptr<Process> createSmallDeformationNonlocalProcess(
+    MeshLib::Mesh& mesh,
+    std::unique_ptr<ProcessLib::AbstractJacobianAssembler>&& jacobian_assembler,
+    std::vector<ProcessVariable> const& variables,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters,
+    unsigned const integration_order,
+    BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{prj__processes__process__type}
+    config.checkConfigParameter("type", "SMALL_DEFORMATION_NONLOCAL");
+    DBUG("Create SmallDeformationNonlocalProcess.");
+
+    // Process variable.
+
+    //! \ogs_file_param{prj__processes__process__SMALL_DEFORMATION_NONLOCAL__process_variables}
+    auto const pv_config = config.getConfigSubtree("process_variables");
+
+    auto per_process_variables = findProcessVariables(
+        variables, pv_config,
+        {//! \ogs_file_param_special{prj__processes__process__SMALL_DEFORMATION_NONLOCAL__process_variables__process_variable}
+         "process_variable"});
+
+    DBUG("Associate displacement with process variable \'%s\'.",
```

Comment:
Why is the symbol ' escaped?

Reply:
Just to be on the safe side ;)
 Removing.